Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -660,9 +663,18 @@ void GraphicsContextGLANGLE::prepareTexture()
         // Blit m_preserveDrawingBufferTexture into m_texture.
         ScopedGLCapability scopedScissor(GL_SCISSOR_TEST, GL_FALSE);
         ScopedGLCapability scopedDither(GL_DITHER, GL_FALSE);
-        GL_BindFramebuffer(GL_DRAW_FRAMEBUFFER_ANGLE, m_preserveDrawingBufferFBO);
-        GL_BindFramebuffer(GL_READ_FRAMEBUFFER_ANGLE, m_fbo);
-        GL_BlitFramebufferANGLE(0, 0, m_currentWidth, m_currentHeight, 0, 0, m_currentWidth, m_currentHeight, GL_COLOR_BUFFER_BIT, GL_NEAREST);
+        if (m_useBlitFallback) {
+            GL_BindFramebuffer(GL_FRAMEBUFFER, m_fbo);
+            GLint texture2DBinding = 0;
+            GL_GetIntegerv(GL_TEXTURE_BINDING_2D, &texture2DBinding);
```

Comment:
glPixelStore affects glCopyTexImage2D. I think you would need to ensure consistent pixel store?